docs/platform/porting/graphics-and-ui.md

```diff
@@ -379,6 +416,47 @@ The output backend interface is mandatory. For more information, see [tdm_backen
 | `output_create_capture()`     | Creates a capture object of an output object. The backend module does not need to implement this function if the hardware does not have a capture device. | No  |
 | `output_set_status_handler()` | Sets an output connection status handler. The backend module must call the output status handler when the output connection status has been changed to let the TDM frontend know of the change. | No  |
 | `output_set_dpms_handler()`   | Sets an output DPMS handler. The backend module must call the output DPMS handler when the output DPMS has been changed to let the TDM frontend know of the change. | No  |
+| `output_get_hwc()`            | Gets a hwc object of an output object. The backend module returns the hwc object when the output has TDM_OUTPUT_CAPABILITY_HWC. | Yes  |
+
+The HWC backend interface is mandatory. For more information, see [tdm_backend.h](https://review.tizen.org/gerrit/gitweb?p=platform/core/uifw/libtdm.git;a=tree;h=refs/heads/tizen;hb=refs/heads/tizen).
+
+**Table: HWC backend interface functions**
+
+| Function                   | Description                              | Mandatory          |
+| -------------------------- | ---------------------------------------- | --------- |
+| `hwc_create_window()`                  | Creates a new window on the given HWC. The backend module must implement `hwc_create_window()`. In addition, the backend module creates a private `tdm_hwc_window` and returns its handle. | Yes |
+| `hwc_get_video_supported_formats()`    | Gets the video supported format array for the hwc windows of a hwc object. | Yes  |
+| `hwc_get_video_available_properties()` | Gets the available video property array of a hwcobject. The backend returns the video properties that are predefined in the backend module. | Yes  |
```

Comment:
Please change "hwcobject" to "hwc object".

Consider the same for line number 431.